.github/workflows/unit-tests.yml

```diff
@@ -81,7 +81,7 @@ jobs:
   # https://github.com/marketplace/actions/alls-green#why
   # https://github.com/orgs/community/discussions/33579
   success:
-    name: Tests successful
+    name: Unit Tests successful
```

Comment:
Nit: should probably lower case the "T": "Unit tests successful"

Reply:
Updated.